configs/next/deb/monolithic/focal_dh_change_dest.patch

```diff
@@ -1 +1 @@
-../../../15.0/deb/monolithic/focal_dh_change_dest.patch
\ No newline at end of file
+../../../16.0/deb/monolithic/focal_dh_change_dest.patch
```

Comment:
Are we still going to support Ubuntu Focal for AT 16? If not, I suspect we don't need this file.

Reply:
Good catch. I forgot to remove it.